File: spring-pulsar-samples/java-sample-with-exception-handlers/src/main/java/com/intuit/spring/pulsar/sample04/SampleApp.java
package com.intuit.spring.pulsar.sample04;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {"com.intuit.spring.pulsar.sample04", "com.intuit.spring.pulsar.client"})
public class SampleApp {
    public static void main(String[] args) {
        SpringApplication.run(SampleApp.class, args);
    }
}

File: spring-pulsar-samples/kotlin-sample-with-exception-handlers/src/main/kotlin/com/intuit/spring/pulsar/sample02/SampleApp.kt
package com.intuit.spring.pulsar.sample02

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication(scanBasePackages = ["com.intuit.spring.pulsar.sample02", "com.intuit.spring.pulsar.client"])
class SampleApp

fun main() {
    SpringApplication.run(SampleApp::class.java)
}